app/models/student_profile.py

```diff
@@ -0,0 +1,29 @@
+from flask import current_app
+from flask_login import AnonymousUserMixin, UserMixin
+from itsdangerous import TimedJSONWebSignatureSerializer as Serializer
+from itsdangerous import BadSignature, SignatureExpired
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from .. import db, login_manager
+
+class StudentProfile(db.Model):
+    __tablename__ = 'students'
```

Comment:
It makes more sense to have __tablename__ = 'student_profiles' just for consistency between our class name and table name.